precompiles/addr/Addr.sol

```diff
@@ -8,7 +8,20 @@ IAddr constant ADDR_CONTRACT = IAddr(
 );
 
 interface IAddr {
+    // Transactions
+    function associateWithGas(
```

Comment:
maybe we can drop the `withGas` since it is implied this tx requires gas to send?

Reply:
Makes sense! Renamed to 'associate'